scripts/test_high_value_setup.py

```
@@ -114,14 +127,18 @@ async def test_high_value_manager():
 
         for job in test_jobs:
             is_hv, category = manager.is_high_value(job["reward"])
-            print(f"Job {job['id']}: ${job['reward']} -> {category if is_hv else 'Standard'}")
+            print(
+                f"Job {job['id']}: ${job['reward']} -> {category if is_hv else 'Standard'}"
+            )
 
         # Test stats
         stats = manager.get_stats()
         print(f"\n📊 Current Stats:")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)